cml/clone-amis.go

```diff
@@ -69,9 +69,10 @@ func main() {
 	ami := imagesRes.Images[0]
 
 	var wg sync.WaitGroup
+	wg.Add(len(regions))
```

Comment:
Out of curiosity, what's the advantage of running `wg.Wait(len(slice))` before the loop instead of running `wg.Wait(1)` inside the loop?